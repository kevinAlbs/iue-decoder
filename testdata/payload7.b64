B0ObNHrTR056kS3MSoY7omkCML5HhwnpVS7w9I8BkoUo16wWK9ito9wVIOYgq0/D5Gd/HtiwEJ3x3Uy1Qkgn6Nb2NrYz1LAKeBdK3/EmeUVOjAkWFXu9SnyCu071b1DRFrKw2eYhetsQILsyNiZfzuS8gJ5W7ArF5AjaVY71eS83kkIBhn39cAXB9dhOhb8hMV+iTJD/jQNGNlw/TMo436e5DspINqJ7ZE4tS/NKlP6F0nV6LW/hWHUe3ClCUhDH/OdkcwrYQ8gMYN9jdT4NCzB+DT8RO6lnV+Y=