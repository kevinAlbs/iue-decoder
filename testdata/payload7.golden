BlobSubtype=7
S_KeyID=439b347ad3474e7a912dcc4a863ba269
OriginalBsonType=2
InnerEncrypted=30be478709e9552ef0f48f01928528d7ac162bd8ada3dc1520e620ab4fc3e4677f1ed8b0109df1dd4cb5424827e8d6f636b633d4b00a78174adff12679454e8c0916157bbd4a7c82bb4ef56f50d116b2b0d9e6217adb1020bb3236265fcee4bc809e56ec0ac5e408da558ef5792f37924201867dfd7005c1f5d84e85bf21315fa24c90ff8d0346365c3f4cca38dfa7b90eca4836a27b644e2d4bf34a94fe85d2757a2d6fe158751edc29425210c7fce764730ad843c80c60df63753e0d0b307e0d3f113ba96757e6
